utils/quick_math.cpp

```diff
@@ -0,0 +1,27 @@
+#include "quick_math.h"
+#include <glm/glm.hpp>
+
+namespace math {
+
+glm::vec3 Truncate(const glm::vec3& vector, float max) {
```

Comment:
Change to `TruncateVector`

Reply:
Actually, I think it would make sense to call it `TruncateVec3` ? That way it immediately is clear to the reader what we deal with. Vector itself is kinda overloaded already (Because STL thought it is a good idea to call their DynamicArray Vector, lol).